content/docs/buildpack-author-guide/create-buildpack/adding-bill-of-materials.md

```diff
@@ -146,28 +224,41 @@ pack build test-ruby-app --path ./ruby-sample-app --buildpack ./ruby-buildpack
 ```
 <!--+- "{{execute}}"+-->
 
-You should then be able to inspect your Ruby app for its Bill-of-Materials via:
+Viewing your bill-of-materials requires first pushing the built image to a container registry and then querying the bill-of-materials.  If you are a `docker` user, we can experiment with an OCI container registry by running an instance of Docker's `registry:2` (as an aside, the standard Docker daemon is a container registry, but is not fully [OCI compliant](https://opencontainers.org/)).
```

Comment:
You should be able to view the sbom from the daemon as well if you passed the --local flag.

Reply:
Thanks. I've updated this and removed all the `registry:2` complexity.